jdi-light-angular-tests/src/test/java/io/github/epam/data/TestData.java

```diff
@@ -0,0 +1,13 @@
+package io.github.epam.data;
+
+
+import io.github.com.entities.Users;
+
+public class TestData {
+    public static Users DEFAULT_USER = new Users().set(c-> {
+        c.firstName = "Long"; c.lastName = "Dinh";
```

Comment:
Maybe add a cosntructor to User class and use it here?

Reply:
I created Test Data using DataClass as per instruction in document. Please have a look: https://jdi-docs.github.io/jdi-light/?java#5-test-data

![image](https://user-images.githubusercontent.com/65541638/85246897-4087be80-b476-11ea-8ee9-f2d71c359d23.png)